website/public/fonts/fonts.css

```
@@ -0,0 +1,131 @@
+  font-style: italic;
+  font-display: swap;
+}
+@font-face {
+  font-family: 'Poppins';
+  src: url("b41d3b25-b9f7-40b2-9018-e900fb230ed9.ttf") format('truetype');
+  font-weight: 700;
+  font-style: normal;
+  font-display: swap;
+}
+@font-face {
+  font-family: 'Poppins';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)